source/CMakeLists.txt

```diff
@@ -364,6 +364,13 @@ target_sources(EndlessSkyLib PRIVATE
 	comparators/ByGivenOrder.h
 	comparators/ByName.h
 	comparators/BySeriesAndIndex.h
+	concurrent/Parallel.h
+	concurrent/PartiallyGuardedList.h
+	concurrent/PartiallyGuardedMap.h
+	concurrent/PartiallyGuardedVector.h
```

Comment:
Can these two be added when they are actually used?
PartiallyGuardedMap and PartiallyGuardedVector
would make more sense to me